dist/cornerstoneTools.js

```diff
@@ -2229,6 +2229,55 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneMath, cornerstoneTo
  
 // End Source; src/imageTools/rectangleRoi.js
 
+// Begin Source: src/imageTools/saveImage.js
+var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
+
+    "use strict";
+
+    if(cornerstoneTools === undefined) {
+        cornerstoneTools = {};
+    }
+
+    function saveAs(element, filename)
+    {
+        var canvas = $(element).find("canvas").get(0);
```

Comment:
Does it make sense to extract `$(element).find("canvas").get(0);` as a helper method?